### PR TITLE
Add Script Commands: search-wpengine

### PR DESCRIPTION
**Pull request by crstauf**
## Description

Add a "Search WP Engine Installs" (by input parameter) script, as well as directly open a WP Engine install's dashboard (by input parameter).

Depending on how complicated the Raycast API is, these may get replaced with an extension, but in the meantime serve quite well.

## Type of change

- [x] New script command

## Screenshot

Screen Shot 2020-12-04 at 10 44 41 PM

## Dependencies / Requirements

- `open`

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/web-searches/wpengine-install.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# @raycast.description Open specified [WP Engine](https://wpengine.com) install.
+
+# @raycast.icon images/wpengine-logo.png
+# @raycast.mode silent
+# @raycast.packageName Web Searches
+# @raycast.schemaVersion 1
+
+# @raycast.argument1 { "type": "text", "placeholder": "Install" }
+
+open "https://my.wpengine.com/installs/${1// /}"
```
Comment (PitNikola) [inline commands/web-searches/wpengine-install.sh:15]:
```suggestion
open "https://my.wpengine.com/installs/${1// /%20}"
```

**Review by PitNikola — COMMENTED** [review]
What do you think about moving it to wordpress folder?

**Comment by crstauf** [comment]
WordPress and WP Engine are very different things, so putting them in the same directory doesn't make sense.

A user may have a WP Engine site, but not be a WordPress developer (and therefore not need the searching scripts).

**Comment by PitNikola** [comment]
Their tagline on the website says "WordPress Hosting, Perfected" so it looked related to WordPress. But you probably know better 🙂

**Pull request merged**